### DISCUSSION TITLE
adapt_mesh not found in pyamg

### DISCUSSION THREAD

**Discussion by danielenriquez59** [su2code/SU2#1469] [Q&A]
I recently downloaded and compiled the feature_adap branch of SU2. 

After installing I attempted to run an adaptation case with pyamg.  However my code errors out with, 

'File "C:\bin\SU2\amginria\interface.py", line 136, in call_pyamg
    mesh_new = pyamg.adapt_mesh(mesh, remesh_options)
AttributeError: module 'pyamg' has no attribute 'adapt_mesh''

Does the newest version of pyamg no longer have this method? If so, what is the fix?

Thanks
Daniel

**Comment by PENGYAN777** [comment]
I meet the same problem, can anybody help? thank you

**Comment by nirvananas** [comment]
Same problem here! I would be interested in the answer!
